### DISCUSSION TITLE
Login Failure in demo app

### DISCUSSION THREAD

**Discussion by jayantsadhu** [realworld-apps/realworld#1553] [DEMO / API health status] [answered]
https://demo.realworld.io/#/login is not working. It is getting failed and coming back to general page.

**Comment by jayantsadhu** [comment]
Although I see REST APIs are working fine

**Reply by geromegrignon** [reply, marked as answer]
Hi, use the one promoted on the GitHub repository: https://conduit.realworld.how/

I do not have control over the old one.

**Reply by jayantsadhu** [reply]
Thanks @geromegrignon for pointing out

**Reply by ngterry** [reply]
I got "Site Not Found" error when trying to visit https://conduit.realworld.how.  Moreover, the "discover our demo" link in https://main--realworld-docs.netlify.app/ is pointing to https://demo.realworld.io/ instead of https://conduit.realworld.how .

**Reply by geromegrignon** [reply]
@ngterry I just fixed it for conduit.realworld.how
For the documentation, I'll delay it as I'm refactoring this repo as I met some issues with Docusaurs as the documentation solution and planning to move to Astro Starlight.

**Reply by geromegrignon** [reply]
@ngterry That's fixed for the docs too (documented as demo.realworld.how whi is an alias for conduit.realworld.how)

**Reply by jayantsadhu** [reply]
Hi @geromegrignon  API response formats has been changed but documentation is still showing same.
for example: https://api.realworld.io/api/articles

**Reply by geromegrignon** [reply]
Hi @jayantsadhu, as an experiment to reduce database egress, I inspected API response format to find out, the original creators included the article body in this endpoint while never used (we use the title/description in the UI but not the body).
Given it could be as large as the rest of the article content, I removed it with great benefits for the egress (and performance).

I'll update the documentation today.

**Reply by geromegrignon** [reply]
Documentation updated
